client/reader/discover/controller.js

```diff
@@ -73,8 +90,27 @@ const exported = {
 		/* eslint-enable wpcalypso/jsx-classname-namespace */
 		next();
 	},
+
+	discoverSsr( context, next ) {
+		const state = context.store.getState();
+		if ( ! isUserLoggedIn( state ) ) {
+			context.renderHeaderSection = renderHeaderSection;
+		}
+		const selectedTab = context.query.selectedTab || DEFAULT_TAB;
+		const tabTitle = getSelectedTabTitle( selectedTab ) || '';
+		/* eslint-disable wpcalypso/jsx-classname-namespace */
```

Comment:
Feel free to get rid of those `eslint-disable` rules in this file. Seems like since #66137 the rule only checks for following the BEM convention.

Reply:
Good to know; I've removed the rules.